odk1-src/briefcase-using.rst

```diff
@@ -314,9 +347,11 @@ Exporting forms to CSV
       Optional params for -e operation:
         -em,--exclude_media_export          Exclude media in export
         -end,--export_end_date <arg>        Export end date (inclusive) (yyyy-MM-dd or yyyy/MM/dd)
+        -ig,--include_geojson               Include a GeoJSON companion file with spatial data
```

Comment:
s/companion//

Reply:
We should change the help text on Briefcase end before making this change.